src/Tribe/Main.php

```diff
@@ -615,8 +615,6 @@ public function bind_implementations(  ) {
 
 			if ( ! tribe_events_views_v2_is_enabled() ) {
 				tribe_singleton( 'tec.customizer.widget', new Tribe__Events__Customizer__Widget() );
-			} else {
-				tribe_singleton( 'tec.customizer.text', new Tribe__Events__Customizer__Text() );
 			}
```

Comment:
Is there a way to deprecate tribe_singleton( 'tec.customizer.text', new Tribe__Events__Customizer__Text() );

Pro calls it and if they do not update Core and Pro at the same time it could trigger a fatal. ( if we do not require the latest version )

Reply:
I think your addition of a `_deprecated_file` later is fine.